### DISCUSSION TITLE
如何将 Obsidian 用成拆书（看书）软件

### DISCUSSION THREAD

**Discussion by Quorafind** [obsidianzh/forum#32] [Ideas想法]
思路一：

2020-12-23 更新：[知乎文章](https://zhuanlan.zhihu.com/p/338719525)

目前，Obsidian 只支持 markdown 文件，但是事实上，我们可以将 pdf 的文件转成 txt 后经过一定的处理后，将格式化后的超长文档放在 Obsidian 里面看：

- 利用 Abbyy finedreader 将需要看的文献或书籍批量转成 txt 文本，然后利用 Emeditor 将其中的大纲目录提取到目录单且转成对应 Obsidian 可用的 [[#]] 格式引用大纲；

- 利用 Note Refactor 插件可以将大部头一个个章节拆分成自己所理解的内容，而后利用双链将自己的书评和文本内容联系起来；解释参考：<https://github.com/obsidianzh/forum/discussions/32#discussioncomment-243700>

- 同时利用 Obsidian Mindmap 插件实现简单的大纲脑图功能；

- 用 Obsidian to Anki 实现将文本内容移入 Anki 进行复习的功能；（此处可以更改成使用 Supermemo 直接进行）

- 利用 Obsidian Citation 插件将引证以及考经据典的内容引入相关的引用文献；

最后将上述过程中所写的所有笔记整理成自己的笔记库内容，且放回到 Evergreen 目录中，拆书完毕。

思路二：

目前，Obsidian 的 URI 功能可以随时使用，在外部的编辑器看 Pdf 的时候，将对应的 Obsidian 的笔记链接复制且粘贴到对应的位置，从而实现 pdf + Obsidian 的联系，也是十分方便。

以上是最近发现的一些可能用得上的结合使用技巧，欢迎各位有什么其它看法的可以在后文写一下

**Comment by lillian-who** [comment]
我觉得硬要all-in-one是强app之所难。。。
目前我在用的方案是，电脑上一边看一边划线，直接顺手把划线部分粘贴进obsidian，并且写上章节
这一步完成了摘抄采集
然后是现在在obsidian缺的一部分，书的大纲整理。目前在纸质笔记本上做，好处是能尽量精简。一旦在电子笔记上，由于写入方便，会写很多无用的东西。

感谢楼主让我知道obsidian还有脑图插件

**Reply by Quorafind** [reply]
我也是这样认为的，但是我现在会收集网络短文（直接下载）到自己的 Obsidian 笔记库里面（类似印象笔记），而为了能充分利用上这些笔记，防止重演印象笔记中笔记无法管理无法寻找的状况，我觉得能在 Obsidian 中用简单的“拆书？”功能还是还重要的🤣🤣

**Reply by makerjackie** [reply]
> 电脑上一边看一边划线，直接顺手把划线部分粘贴进obsidian

网页笔记我现在用的是Diigo插件，阅读时高亮+做笔记，然后可以一键复制自己高亮/做笔记的地方到obsidian。不过我一直在寻找不需要科学上网+费用较低的方案，Diigo有时我网络有问题，目前找到一款chrome插件BeaNote，但是导出稍有点不方便

**Reply by lillian-who** [reply]
> 我也是这样认为的，但是我现在会收集网络短文（直接下载）到自己的 Obsidian 笔记库里面（类似印象笔记），而为了能充分利用上这些笔记，防止重演印象笔记中笔记无法管理无法寻找的状况，我觉得能在 Obsidian 中用简单的“拆书？”功能还是还重要的🤣🤣

请问你是怎么下载到obsidian的啊？比如我想把一篇网络文章存下来，暂时没有特别好的方式。如果能够做到转存到obsidian，那么obsidian就更强大了。

**Reply by lillian-who** [reply]
> > 电脑上一边看一边划线，直接顺手把划线部分粘贴进obsidian
> 
> 网页笔记我现在用的是Diigo插件，阅读时高亮+做笔记，然后可以一键复制自己高亮/做笔记的地方到obsidian。不过我一直在寻找不需要科学上网+费用较低的方案，Diigo有时我网络有问题，目前找到一款chrome插件BeaNote，但是导出稍有点不方便

握个手，希望各大阅读的软件，包括网页阅读，都能够提供markdown格式的复制粘贴。
我现在大多阅读是在微信读书，一键复制划线笔记，印象笔记就支持的很好，粘贴进去格式非常好看。但是除了印象笔记，其他的笔记软件支持都不好，粘贴进去格式都不见了。

**Reply by Quorafind** [reply]
> > > 电脑上一边看一边划线，直接顺手把划线部分粘贴进obsidian
> > 
> > 
> > 网页笔记我现在用的是Diigo插件，阅读时高亮+做笔记，然后可以一键复制自己高亮/做笔记的地方到obsidian。不过我一直在寻找不需要科学上网+费用较低的方案，Diigo有时我网络有问题，目前找到一款chrome插件BeaNote，但是导出稍有点不方便
> 
> 握个手，希望各大阅读的软件，包括网页阅读，都能够提供markdown格式的复制粘贴。
> 我现在大多阅读是在微信读书，一键复制划线笔记，印象笔记就支持的很好，粘贴进去格式非常好看。但是除了印象笔记，其他的笔记软件支持都不好，粘贴进去格式都不见了。

试一下简悦？这个插件我一直在用，还是很方便的，现在还支持指定导出的位置，价格也算便宜

**Reply by maybemerald** [reply]
> > 我也是这样认为的，但是我现在会收集网络短文（直接下载）到自己的 Obsidian 笔记库里面（类似印象笔记），而为了能充分利用上这些笔记，防止重演印象笔记中笔记无法管理无法寻找的状况，我觉得能在 Obsidian 中用简单的“拆书？”功能还是还重要的🤣🤣
> 
> 请问你是怎么下载到obsidian的啊？比如我想把一篇网络文章存下来，暂时没有特别好的方式。如果能够做到转存到obsidian，那么obsidian就更强大了。

可以试试简悦，相当方便

**Comment by cirvoe** [comment]
我尝试过把epub转成md，最不好办的是图片，我不会编程，而是用转换工具，转成docx，docx可以解压，可以得到一个装图片的文件夹，之后把docx转成md，运气好的话就可以获得显示图片的md了，途中可能会碰上编码问题或图片重复的问题。如果是电子版pdf也可以这么做。

**Reply by Quorafind** [reply]
等后续看看有没有更好的工作流，这种方式不方便，不过我现在应用这个都是用在一些简短文章和文献上，真正在大部头上这样做还没试过，欢迎分享你的经验

**Comment by cyddgh** [comment]
期待具体步骤的更新。我今天也了解了Note Refactor，看了上文中的步骤，感觉很有趣。
但在此也说一点我目前遇到的困惑：知识点链接和打断阅读状态。①在拆书的过程中产生的知识点，应该会归属于2个地点，一个是客观上的归属，例如这知识点A、B、C依次来自于某本书，那ABC的排序就是固定的。另一个是主观上的归属，知识点A可能在页面1上，B在页面2，C在页面3上。目前的流程要同时安排两个方向很是困难，操作比较繁琐。②能完全OCR的精品书还是少的，很多材料需要手动输入，而且还要输入页面，在实际操作中也不轻松，可能打断了静心阅读的状态。

**Reply by Quorafind** [reply]
完全无错误的 OCR 的确不可，但是可以将文章完全 OCR 还是可以的。

我认为，第一个问题的解决方案正好就是链接以及zk卡的优势所在，假设你考虑使用我上述所说的方式进行拆书：

> 利用 Note Refactor 插件可以将大部头一个个章节拆分成自己所理解的内容，而后利用双链将自己的书评和文本内容联系起来；

这里的意思并不是说第一次就用 NR 将文本中的知识点分类拆了出来，只是让你将其中的章节分开，乃至于将具体的文本内容分别拆出来，这只是泛读（甚至泛读都算不上），仅是为了让你将某个内容初步按照原来的书籍的顺序分成一系列的章节卡片，这是其一——但是泛读的时候也可以使用精读的技巧进行，来实现将书本读清楚的实现：

接下来，你按照自己的章节分别开始阅读，然后真正对里面遇到的具体知识点A、B、C用自己想到的解释方式进行拆，即遇到知识点A，将其从章节卡片中拆出来（这不就是ZK嘛），然后接下来就是引入费曼技巧的一步：用自己的话复述一次，此时，有两个整理技巧

其一是将自己复述的话直接变成章节卡片中的内容，然后引用利用 NR 抽掉的原文；其二则是简述抽掉的原文，也在新的卡片（抽出来）中加上对这个笔记看法，即书评 

遇到知识点 A 假设你将它拆出来了，然后将其按照你理解的那个主观归属的A归类成你理解的主观归属的文件名，例如要处理的内容：

[[归类1-知识点A|知识点A处于当前的书籍中的内容]]，没错，也是需要用上别名来处理这件事，然后继续拆，拆到你遇到的同样的[[归类1--知识点B|知识点B处于当前的书籍中的内容]]，因为实际产出的卡片是单独的，并不是连续的，即使你真的弄了一百张整本书的拆书卡片，其中的知识点也是分开的，你就不会拘泥于原书的顺序。

然后，最后，在你完全将书本分成N张卡片后，再大量使用块引用或者直接使用 TEXT expander 命令将该书的所有卡片引用到某张卡片中，这样，你就拥有了自己的 TOC 或者说 简略 MOC

这样你的笔记结构应该是：

[[原始书籍]]-->[[章节卡片]]-->[[知识卡片]]-->[[自己整合的卡片]]

最后，你产出的就是一个属于自己的目录表，而且能够在知识卡片中自由引用新的文献，更重要的是，能看到原来的文献，最后形成属于自己的知识体系。

此外，如果你觉得使用 NR 比较麻烦，有另一个叫做 workbench 的插件，也可以在看书的过程中将笔记抽离出来，最后粘贴（适合短文）；还有一个 Highlight Extract 的插件，可以将高亮的文本抽离出来，都是一种拆书的方式。

上述的拆书方式实践起来，只要熟练了，理论上是无比畅快（前提是喜欢看书）

**Comment by 23784148** [comment]
嗨，我是 [简悦](http://simpread.pro/) 的作者，同样也是 Obsidian 的使用者。我提供一个目前在使用的全流程方案。

> 简悦 2.1 + 同步助手（导出到任意目录功能） + Obsidian  方式。

### 思路如下

1. 在网页使用简悦的标注进行快速标注（简悦支持连续标注），而这个过程类似拆书。
2. 之后将这些快速标注的内容通过 [简悦 · 同步助手](http://ksria.com/simpread/docs/#/Sync) 导出到 Obsidian 的目录（以 Markdown 的形式）
3. 同时在标注时，也可以做一些备注，标签的操作（如果只是快速拆书的话，这些可后期再做），甚至于可以将某条标注单独导出。
4. 快速标注后，会在简悦的稍后读生成一个基于标注的页面，在这个页面可以集中管理这些标注，以便做二次修改（补充），同时这些内容也可以导出为 Markdown
5. 最后每天通过这种方式产生的稍后读（标注）会已邮件汇总的方式发给你指定的邮箱（需配置），用于当天的 [加深回顾](http://ksria.com/simpread/docs/#/%E6%AF%8F%E6%97%A5%E5%9B%9E%E9%A1%BE)。
6. 这篇稍后读（包含如干拆书标注）也可以通过 发送邮件 / 发送到 Kindle / 导出的 PDF 的形式 / 导出 Epub 的形式来辅助你对这篇文章的加深记忆。

### 相比与 Diigo

> 因为上面有用户提到了 Diigo ，所以基于 Obsidian 的使用场景来简单的对比描述下。

1. 简悦支持单条标注导出，所以省去了 **复制/粘贴** 的操作。

2. 简悦的数据文件都在本地（稍后读和配置文件），所以与 Obsidian 天然搭配。

3. 简悦 + 坚果云 + Obsidian 的同步方案，可以实现：你的数据你掌握，且放在你需要的地方，进行全平台同步。

***

因为很喜欢 Obsidian 所以多说点方法论的事情。  😀 

> 一般来说使用 Obsidian 都是为了补齐 PKM 中的最后一环： **内化**

而 KPM 的前几个环节：信息的收集 和 信息的过滤（有效信息）我一般都使用了下面的方案：

1. 使用 Inoreader 过滤等功能来筛除一批无效的噪音。

2. 留下来的自然是需要的内容（信息）。

  > 通过简悦 2.1 提供的 [RSS 阅读器辅助方式](http://ksria.com/simpread/docs/#/RSSReader) 或 [此方式](https://github.com/Kenshin/simpread/discussions/1559) 将这类信息直接导入到简悦的稍后读。

3. 这些信息（还不能称之为知识），大致包括如下几类：

    - 只是个人感兴趣的信息，但不具有反复理解的内容。

      > 我会把它们通过简悦发送到邮件，利用碎片时间在手机上阅读（阅读后会删除它，因为只是用来阅读。（当然，你觉得有意义的话，可以使用离线 HTML 的方式，将它永久的保存下来。

   - 有意义的信息

      > 通过上面描述的拆书方式，发送到 Obsidian 做以后 **内化** 用（内化的结果是：将这些信息转换为你的知识

    - 零碎但具有收集意义的信息

      > 比如：我要去旅行，再这个前提下，会收集旅行相关的内容，这些内容并不具有知识的属性，但确实是你需要的内容，且它们都分散在不同的页面。

      > 遇到这种情况的话，我会通过简悦 [合辑的功能](http://ksria.com/simpread/docs/#/%E7%A8%8D%E5%90%8E%E8%AF%BB?id=%e5%90%88%e8%be%91) 将有需要的内容整理到一个 **页面**，[这是一个合辑的例子](https://simpread.pro/s/z2Axt6i1)

      > 同样即便是合辑，也支持稍后读的全部操作，如：导出 Markdown 等等

***

### 再蹭个热度，使用简悦如何进行**卢曼的卡片笔记法 （Zettelkasten）的分类方法**：

1. 阅读资料，并在过程中高亮或者会随手记下临时笔记 （Fleeting notes）
2. 将高亮的部分或者临时笔记总结为文献笔记 (Literature notes)
3. 用自己的话将文献笔记转述为永久笔记 (Permanent notes)

实现它们的思路：

1. Fleeting notes → 简悦的标注
2. Literature notes → 简悦的合辑（或对某个稍后读的多个标注进行详细备注的操作，简悦的标注与稍后读具有一样的地位和结构）
3. Permanent notes → 使用 Obsidian

***

最后的最后，我也是一个对 PKM GTD 很感兴趣的用户，希望借此也可以多多跟大家交流。
如果有对简悦操作上啥问题，可以随时在这里找我。（ 这个 Github 只是我的小号，我的大号 https://github.com/kenshin/
关于其它简悦的联络方式就不放到这里了，毕竟这里是讨论 Obsidian 的地方。

🙏

**Reply by makerjackie** [reply]
我擦牛逼啊，我试试

---

一开始体验有点懵逼，后面熟悉就好一些了

我期望是类似Diigo的划线标注，但发现需要先进入**阅读模式**后才能标注，这样就无法类似diigo/beanote那样在原始网页上保留高亮痕迹了。此外，使用时稍觉得卡顿，简悦是一个不错的浏览插件，不过为了适配太多生产力工具，界面上比较复杂，学习成本就高一些（update，可以通过[插件](https://github.com/Kenshin/simpread/issues/1613#issuecomment-751332824)简化图标）

使用一段时间后再来追评

**Reply by 23784148** [reply]
欢迎使用。

另外提一句：简悦不是传统的 **稍后读 or 阅读模式** 产品，所以在使用上如果没留意一些信息的话，有可能会产生一点不适感。（也就是对新手来说有一点上手曲线）
当然，这些不适感也包括：设计不合理的地方以及某些隐性 Bug。

但简悦大方向就是这样的，这点不会变。

***

解决办法也很简单，多留意提示信息以及多看些简悦的说明文档以及翻翻 Github Issues（总有前人踩过的坑不是？） 😂

**Reply by 23784148** [reply]
> 这样就无法类似diigo/beanote那样在原始网页上保留高亮痕迹了

目前只能在阅读模式下使用，这么做的原因是：**只有有深度阅读的页面才有做标注的意义**。（所以，从这方面来将也自动剔除了一些不必要的干扰信息源。

**Reply by cyddgh** [reply]
很好的使用介绍，很多ob的用户也都在使用简悦。

**Reply by 23784148** [reply]
@Quorafind 

嗨，近期发现确实也有一些 Obsidian 用户在使用简悦作为导出手段，不知道能否将上面的说明单开一个 discussion 加以说明？

**Reply by Quorafind** [reply]
> @Quorafind
> 
> 嗨，近期发现确实也有一些 Obsidian 用户在使用简悦作为导出手段，不知道能否将上面的说明单开一个 discussion 加以说明？

已经将您的回复开了个新贴，后续请关注那个帖子：<https://github.com/obsidianzh/forum/discussions/38>
